test_communication/CMakeLists.txt

```diff
@@ -135,7 +135,7 @@ if(BUILD_TESTING)
     endif()
     set_tests_properties(
       ${target}${target_suffix}
-      PROPERTIES REQUIRED_FILES "${target_path}/${target}${target_suffix}"
+      PROPERTIES REQUIRED_FILES "$<TARGET_FILE:${target}${target_suffix}>"
```

Comment:
it is obsolete indeed https://github.com/ros2/system_tests/pull/173/commits/d9d9653020658c140752a823ca3974515465de1d